sorl_thumbnail_serializer/fields.py

```diff
@@ -82,12 +84,14 @@ def to_representation(self, value):
 
         image = get_thumbnail(value, self.geometry_string, **self.options)
 
-        try:
-            request = self.context.get('request', None)
-            return request.build_absolute_uri(image.url)
-        except:
-            try:
-                return super(HyperlinkedSorlImageField, self).to_representation(image.url)
-            except AttributeError:  # NOQA
-                return super(HyperlinkedSorlImageField, self).to_native(image.url)  # NOQA
+        request = self.context.get('request', None)
+        assert request is not None, (
```

Comment:
Not sure how this would fare with existing usage as the current implementation attempts to use  `FileField.to_representation()` behaviour (i.e. relative path to image) as a fallback. Up until now I've let `to_representation()` fail silently (which admittedly is not always the best option) but this introduces a potential `AssertionError`.

While I'm not too afraid of breaking changes at this stage I think it's worth some consideration. Perhaps this behaviour should only be triggered by a specific option?